src/virtual_memory.c

```diff
@@ -39,6 +39,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #  include <pthread.h>
 #  include <sys/utsname.h>
 #  include <stdio.h>
+#  if defined(MAC_OS_VERSION_11_0) && MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_VERSION_11_0
```

Comment:
I think we should drop this chunk and just use `__builtin___clear_cache()` like jit_compiler_a64.cpp does.